antiplag/nlp/text_preprocessing.py

```diff
@@ -14,7 +14,17 @@
 import string
 from pycontractions import Contractions
 from textblob import TextBlob
+import pytesseract
+from PIL import Image
```

Comment:
I see you're importing pillow (or its slightly modified version - PIL) here, but it's not in requirements. Does it come with django by default or do we need to add it to requirements?